src/index.css

```diff
@@ -16,21 +16,17 @@ body {
   width: 100%;
 }
 
-.category-icon{
+.category-list_icon{
+  fill: #616161;
```

Comment:
I think we could split this CSS file, because that's in the global scope. We should add these props to the component scss module file.